api/src/main/resources/liquibase.xml

```diff
@@ -4,6 +4,345 @@
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+    <changeSet author="slubwama" id="patientflags-1554735240623-1">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="patientflags_displaypoint"/>
+            </not>
+        </preConditions>
+        <comment>Create Table patientflags_displaypoint</comment>
+        <createTable tableName="patientflags_displaypoint">
+            <column autoIncrement="true" name="displaypoint_id" type="int">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="name" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="description" type="VARCHAR(1000)"/>
+            <column defaultValueNumeric="0" name="creator" type="int">
+                <constraints nullable="false"/>
+            </column>
+            <column name="date_created" type="datetime">
+                <constraints nullable="false"/>
+            </column>
+            <column name="changed_by" type="int"/>
+            <column name="date_changed" type="datetime"/>
+            <column defaultValueBoolean="false" name="retired" type="boolean">
+                <constraints nullable="false"/>
+            </column>
+            <column name="retired_by" type="int"/>
+            <column name="date_retired" type="datetime"/>
+            <column name="retire_reason" type="VARCHAR(255)"/>
+            <column name="uuid" type="CHAR(38)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="slubwama" id="patientflags-1554735240623-2">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="patientflags_flag"/>
+            </not>
+        </preConditions>
+        <comment>Create Table patientflags_flag</comment>
+        <createTable tableName="patientflags_flag">
+            <column autoIncrement="true" name="flag_id" type="int">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="name" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="criteria" type="VARCHAR(5000)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="message" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="enabled" type="TINYINT(3)">
```

Comment:
In some places i see "TINYINT" while others i see "int". I think it would look cleaner if you consistently decided to either use upper or lower case.

Reply:
Also, could there be a reason why you are not consistently using boolean for boolean types?

Reply:
@dkayiwa done with this comment